src/pages/auth/register.tsx

```diff
@@ -140,6 +155,143 @@ const Page = () => {
+              <Field
+                as={Select}
+                label="Country"
+                name="country"
+                fullWidth
+                onChange={(_: SelectChangeEvent, element: ReactElement) => {
+                  const country = element.props.value;
+                  setFieldValue("country", element.props.value);
+
+                  if (country == "US") {
+                    setFieldValue("useCase", UseCase.EmbeddedFinance);
+                  } else {
+                    setFieldValue("useCase", UseCase.ExpenseManagement);
+                  }
+                }}
+              >
+                <MenuItem value="AT" disabled>
+                  Austria
+                </MenuItem>
+                <MenuItem value="BE" disabled>
+                  Belgium
+                </MenuItem>
+                <MenuItem value="HR" disabled>
+                  Croatia
+                </MenuItem>
+                <MenuItem value="CY" disabled>
+                  Cyprus
+                </MenuItem>
+                <MenuItem value="EE" disabled>
+                  Estonia
+                </MenuItem>
+                <MenuItem value="FI" disabled>
+                  Finland
+                </MenuItem>
+                <MenuItem value="FR" disabled>
+                  France
+                </MenuItem>
+                <MenuItem value="DE" disabled>
+                  Germany
+                </MenuItem>
+                <MenuItem value="GR" disabled>
+                  Greece
+                </MenuItem>
+                <MenuItem value="IE" disabled>
+                  Ireland
+                </MenuItem>
+                <MenuItem value="IT" disabled>
+                  Italy
+                </MenuItem>
+                <MenuItem value="LV" disabled>
+                  Latvia
+                </MenuItem>
+                <MenuItem value="LT" disabled>
+                  Lithuania
+                </MenuItem>
+                <MenuItem value="LU" disabled>
+                  Luxembourg
+                </MenuItem>
+                <MenuItem value="MT" disabled>
+                  Malta
+                </MenuItem>
+                <MenuItem value="NL" disabled>
+                  Netherlands
+                </MenuItem>
+                <MenuItem value="PT" disabled>
+                  Portugal
+                </MenuItem>
+                <MenuItem value="SK" disabled>
+                  Slovakia
+                </MenuItem>
+                <MenuItem value="SI" disabled>
+                  Slovenia
+                </MenuItem>
+                <MenuItem value="ES" disabled>
+                  Spain
+                </MenuItem>
+                <MenuItem value="GB" disabled>
+                  United Kingdom
+                </MenuItem>
+                <MenuItem value="US">United States</MenuItem>
+              </Field>
+              <Divider />
+              <FormLabel>What kind of app are you building?</FormLabel>
```

Comment:
This looks cool but I'm having second thoughts about exposing this selector. The user story here is that when someone is registering, they are pretending to be a connect account admin. Whereas "What kind of app you are building?" is a question for the dev. I think for now, we should remove this selector and use the country to hide/show different versions of this text below. I'm imagining how a potential startup could launch using this sample app in US and UK at the same time. They'd offer whatever use case is available in that region without the user selecting it. As for sending the use case value to the backend, could we for now just derive the it from the country?
 
![CleanShot 2024-01-17 at 23 45 56](https://github.com/stripe-samples/issuing-treasury/assets/103917180/bb40ceb0-590a-4249-819e-03af9d2f04bf)

Reply:
Okay I think we _could_ keep the selector but I'd change the phrasing so that it is more CA admin facing. So instead of showing "Embedded Finance", I think it would be more CA admin friendly to describe what they'd be able to do: "Full-stack financial services to create cards, make payments, and make and receive payments with a financial account". And similarly for "Expense management" it could just be "A commercial pre-funded card issuing program". What do you think? We should still conditionally change the above screenshot text regardless. It can expand on this selector's text.

Reply:
Yeah, makes sense. I didn't feel _great_ about the copy in that selector, and making it more specific will help.

It's worth noting that even though this is presented as a radio button selector, because the use cases are (for now, at least) mutually-exclusive with countries, it's more informational than anything. I think it's useful in that sense because it sets an expectation that the experience will be different in _some way_.

Reply:
I'm also very open to suggestions for a better term than "app" - it's a bit reductive, but I can't find any better term that is more concise that means "Issuing/Treasury Setup"

"_What kind of BaaS product are you building?_" maybe?

Reply:
Okay so to avoid more back and forth for now and to unblock the rest of your PRs, I made some changes to this section of the branch. We can discuss it more this week as I don't think the copy is final.